disable_odoo_online/models/publisher_warranty_contract.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013 Therp BV (<http://therp.nl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class publisher_warranty_contract(models.AbstractModel):
+    _inherit = 'publisher_warranty.contract'
+
+    @api.multi
+    def update_notification(self, cron_mode=True, context=None):
```

Comment:
As suggested by Odoo stuff, we made similar feature in [web_debranding](https://www.odoo.com/apps/modules/9.0/web_debranding/) optional, but don't allow to switch it off (i.e. we always send request) for enterprise users.

Code the check whether it enterprise:

    from odoo.release import version_info
    is_enterprise = version_info[5] == 'e'